server/controllers/auth/login/get-user-id.js

```diff
@@ -14,7 +14,7 @@ module.exports = async (req, res, next) => {
       const value = await verifyToken(token, SECRET_KEY);
       res.json({ id: value.userId });
     } else {
-      res.status(401).json({ message: 'You are not registred yet' });
+      res.json({ id: 0 });
```

Comment:
why to send status of 200 while the user is sending a bad request

Reply:
this is not a request from the user